qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -110,93 +118,245 @@ def __init__(self, data, validate=True):
-        return f"Clifford({repr(self.table)})"
+        return f"Clifford({repr(self.paulis)})"
 
     def __str__(self):
-        return "Clifford: Stabilizer = {}, Destabilizer = {}".format(
-            str(self.stabilizer.to_labels()), str(self.destabilizer.to_labels())
-        )
+        labels = self.paulis.to_labels()
+        num_qubits = self.num_qubits
+        stabilizer = slice(num_qubits, 2 * num_qubits)
+        destabilizer = slice(0, num_qubits)
+        return f"Clifford: Stabilizer = {labels[stabilizer]}, Destabilizer = {labels[destabilizer]}"
 
     def __eq__(self, other):
         """Check if two Clifford tables are equal"""
-        return super().__eq__(other) and self._table == other._table
+        return super().__eq__(other) and self._paulis == other._paulis
 
     # ---------------------------------------------------------------------
     # Attributes
     # ---------------------------------------------------------------------
     def __getitem__(self, key):
         """Return a stabilizer Pauli row"""
-        return self._table.__getitem__(key)
+        return self._paulis.__getitem__(key)
 
     def __setitem__(self, key, value):
         """Set a stabilizer Pauli row"""
-        self._table.__setitem__(key, value)
+        self._paulis.__setitem__(key, value)
 
     @property
+    def paulis(self):
+        """Return PauliList"""
+        return self._paulis
+
+    @paulis.setter
+    def paulis(self, value):
+        """Set the Pauli list"""
+        if not isinstance(value, PauliList):
+            value = PauliList(value)
+        self._paulis._z[:, :] = value._paulis._z
+        self._paulis._x[:, :] = value._paulis._x
+        self._paulis._phase[:] = value._paulis._phase
+
+    @property
+    @deprecate_function(
+        "The Clifford.table method is deprecated as of Qiskit Terra 0.19.0 "
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def table(self):
         """Return StabilizerTable"""
-        return self._table
+        return StabilizerTable(self.tableau(), phase=self.paulis.phase // 2)
 
     @table.setter
+    @deprecate_function(
+        "The Clifford.table method is deprecated as of Qiskit Terra 0.19.0 "
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def table(self, value):
         """Set the stabilizer table"""
         # Note this setter cannot change the size of the Clifford
         # It can only replace the contents of the StabilizerTable with
         # another StabilizerTable of the same size.
         if not isinstance(value, StabilizerTable):
             value = StabilizerTable(value)
-        self._table._array[:, :] = value._table._array
-        self._table._phase[:] = value._table._phase
+        self.paulis(value)
```

Comment:
I'm fairly sure `self.paulis(value)` is not correct here.  Perhaps you were thinking of `self.paulis = value`?

Reply:
My bad. You are right.